linode/sshkey/framework_resource.go

```diff
@@ -62,10 +70,14 @@ func (r *Resource) Read(
 	req resource.ReadRequest,
 	resp *resource.ReadResponse,
 ) {
+	tflog.Debug(ctx, "Create linode_sshkey")
+
 	client := r.Meta.Client
 
 	var data ResourceModel
 
+	ctx = populateLogAttributes(ctx, data)
```

Comment:
Because this resource has been migrated to use framework, I think we can just use the build-in function to populate log attributes like:
```suggestion
	ctx = tflog.SetField(ctx, "sshkey_id", id)
```